### PR TITLE
Cleanup 3d-tiles traversal

### PR DESCRIPTION
**Pull request by xintongxia**

### CODE REVIEW

File: modules/3d-tiles/src/lib/tileset/tileset-3d-traverser.js [hunk 1]
```diff
@@ -63,57 +63,16 @@ export default class Tileset3DTraverser {
...
-    if (!this.options.skipLevelOfDetail) {
-      if (tile.contentAvailable) {
-        // The tile can be selected right away and does not require traverseAndSelect
-        this.selectTile(tile, frameState);
-      }
-      return;
-    }
-
-    // If this tile is not loaded attempt to select its ancestor instead
-    const loadedTile = tile.contentAvailable ? tile : tile._ancestorWithContentAvailable;
```
Comment (ibgreen) [inline modules/3d-tiles/src/lib/tileset/tileset-3d-traverser.js]: @loshjawrence Don't we need this logic, to show parents when children are not loaded?

File: modules/3d-tiles/src/lib/tileset/tileset-3d-traverser.js [hunk 2]
```diff
@@ -314,7 +273,7 @@ export default class Tileset3DTraverser {
       let refines = false;
 
       if (this.canTraverse(tile)) {
-        refines = this.updateAndPushChildren(tile, stack, frameState) && parentRefines;
```
Comment (xintongxia) [inline modules/3d-tiles/src/lib/tileset/tileset-3d-traverser.js]: @loshjawrence correct me if I am wrong, it seems not necessary to check `&& parentRefines `here.

Comment (loshjawrence) [reply modules/3d-tiles/src/lib/tileset/tileset-3d-traverser.js]: You want the parent refinment status to trickle down to the descendants. `refines` gets assigned to `tile._refines` at the end.

File: modules/3d-tiles/src/lib/tileset/tileset-3d-traverser.js (hunk 1 above)
Comment (loshjawrence) [reply modules/3d-tiles/src/lib/tileset/tileset-3d-traverser.js]: This logic is for skipLOD which requires issuing stencil gpu commands to get it to look not terrible.

Comment (ibgreen) [reply modules/3d-tiles/src/lib/tileset/tileset-3d-traverser.js]: OK in that case, let's cut for now.

**Pull request closed without merging**